src/sentry/codecov/endpoints/repository/serializers.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+import sentry_sdk
+from rest_framework import serializers
+
+logger = logging.getLogger(__name__)
+
+
+class RepositorySerializer(serializers.Serializer):
```

Comment:
A question for the crew:

So we'd originally thought of following a resource based approach; we already have a RepositoriesSerializer, which in theory should just be the plural of a repository. In this PR we're defining a separate serializer, RepositorySerializer, that all of the sudden has a different definition for that same resource. This is technically breaking that pattern, unless we consider a Repository different than Repositories. 

Thoughts?